docs/src/commands/build.md

```diff
@@ -32,6 +32,16 @@ wasm-pack build --out-dir out
 The above command will put your build artifacts in a directory called `out`, instead
 of the default `pkg`.
 
+## Output file names
```

Comment:
We might add some more info on exactly what files are outputted and what the names by default are and what parts of the name change when you use this command.

Reply:
Have no idea how to choose the title here ((

Reply:
@ibaryshnikov maybe "Generated file names"?

Reply:
@ashleygwilliams thank you!